ee/tables/jwt/table.go

```diff
@@ -0,0 +1,195 @@
+package kolide_jwt
+
+import (
+	"context"
+	"crypto"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"io"
+	"log/slog"
+	"os"
+	"strings"
+
+	"github.com/golang-jwt/jwt/v5"
+	"github.com/kolide/launcher/ee/dataflatten"
+	"github.com/kolide/launcher/ee/tables/dataflattentable"
+	"github.com/kolide/launcher/ee/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+// I've set 3 different states for if the signature is verified:
+// VALID - The token parsed without errors and the signature was successfully validated.
+// INVALID - The signature attempted to validate with the matched public key, but it was a bad key.
+// UNKNOWN - The default state. This can mean that no key id matched, or simply no keys were provided to validate against.
+const (
+	Valid   = "VALID"
+	Invalid = "INVALID"
+	Unknown = "UNKNOWN"
+)
```

Comment:
Thinking about this a little more, I think we'll want to differentiate between something that's been validated on the client. and something that's been validated on the server. Maybe:
```
const (
	validString   = "client"
	invalidString = "invalid"
	unknownString = ""
)
```

Then the server can put "server" in? Or `client validates` and `server validated`? Not sure.
(I also think unknown should be a blank string, or even NULL, And that these shouldn't be exported)

Reply:
How do you feel about the states of:

```
	Valid   = "client_valid"
	Invalid = "client_invalid"
	Unknown = "client_unverified"
```

I don't really have a preference here, but am wondering if that feels a bit better? Would you still prefer to have unknown be blank/nil?

Reply:
I don't think we need `client` in the invalid or unknown case. Invalid is invalid. the case I want to enable is a very high confidence that something that was validated serverside, really was. 

I do think unknown should be nil. It feels more like SQL

Reply:
Should I remove the unknown constant and put `nil` as the default, or leave the `Unknown` const as an empty string?

Reply:
I'd probably omit `verified` completely. It'll come through as nil.

Reply:
Sounds good.